test/post-911-gib-status/00-basic-flow.e2e.spec.js

```diff
@@ -0,0 +1,33 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts.js');
+const GibsHelpers = require('../e2e/post-911-gib-status-helpers.js');
+const LoginHelpers = require('../e2e/login-helpers');
+const AccountCreationHelpers = require('../e2e/account-creation-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const token = LoginHelpers.getUserToken();
+
+    GibsHelpers.initApplicationMock(token);
+
+    AccountCreationHelpers.initMHVTermsMocks(token);
+
+    // Test flow for unauthed and LOA1 users
+    LoginHelpers.testUnauthedUserFlow(client, '/education/gi-bill/post-9-11/ch-33-benefit');
```

Comment:
I would separate out the unauthed and authed flow into 2 separate tests.

The unauthed test ca probably be nearly identical to this: https://github.com/department-of-veterans-affairs/vets-website/blob/master/test/disability-benefits/e2e/00.unauthed-user.e2e.spec.js

Then this test can do 
```
LoginHelpers.logIn(token, client, '/education/gi-bill/post-9-11/ch-33-benefit', 3)
```
(or whatever the URL currently is)